pkg/util/dbterror/ddl_terror.go

```diff
@@ -531,4 +531,5 @@ var ReorgRetryableErrCodes = map[uint16]struct{}{
 var ReorgRetryableErrMsgs = []string{
 	"context deadline exceeded",
 	"requested lease not found",
+	"create TSO stream failed",
```

Comment:
I don't want to hide the problem for PD client 🤔 We have fixed at caller for many times and it delayed the real fix at PD client side.